src/relay/pushservice/pushservice.py

```diff
@@ -153,7 +153,7 @@ def _build_data_prop(event: Event):
         if event.direction == "received":
             data = _get_data_prop_dict(event)
     elif isinstance(event, MessageEvent):
-        if event.type == "PaymentRequest":
+        if event.type in ["PaymentRequest", "PaymentRequestDecline"]:
```

Comment:
Are we sure that's the two only event type we want to build data for?
Why do we not build data for every event type no matter what they are?

Reply:
Yes currently they should be the only ones. The remaining message types are for example `PaymentMessage` and `RemoteNameMessage`, which are currently not handled as push notifications.